docs/using.rst

```diff
@@ -42,6 +42,11 @@ To install and run the client you just need to type:
 
    ./letsencrypt-auto
 
+.. note:: Let's Encrypt requires Python 2.6+. Check Python_'s website
```

Comment:
This actually isn't quite accurate. Currently, we only support  Python 2.6 and Python 2.7.

The `acme` module supports Python 2.6+, including Python 3+, but Python 3 is not currently supported by the rest of the client,.